testeng/resources/cleanup-python-code-create-pr.sh

```diff
@@ -73,10 +67,25 @@ The following packages were installed:
 \`$PACKAGES\`
 EOF
 )"
+  # if present and non-empty, set the commit message from .git/cleanup-python-code-commit-message
+  # this will allow jobs using this script to make custom commit messages
+  if [ -s "$repo_dir/.git/cleanup-python-code-commit-message" ]
+  then
+    local commit_msg=$(cat "$repo_dir/.git/cleanup-python-code-commit-message")
+  else
+    local commit_msg=$description
```

Comment:
```suggestion
    local commit_msg="$description"
```
I think that without quotes, this will convert all whitespace runs into single spaces.